starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -52,4 +52,13 @@ public void findAll() {
     List<Inventory> actualInventory = this.inventoryDAO.findAll();
     Assert.assertFalse(actualInventory.isEmpty());
   }
+
+  @Test
+  public void create(){
```

Comment:
Checking that there are the correct amount of inventory objects exist within the collection is a good start, however, it would probably help if you also did some additional checks to make sure that the data looks correct under different scenarios. For example, it would be a good idea to set the id of the inventory object before running the create method and then check to see if the id is correct when you run findAll()

Reply:
Added some more tests to the method.